packages/framework/fluid-static/src/fluidContainer.ts

```diff
@@ -68,7 +73,22 @@ import { RootDataObject } from "./rootDataObject";
  * ```
  */
 export interface IFluidContainerEvents extends IEvent {
-    (event: "connected" | "dispose" | "disconnected" | "saved" | "dirty", listener: () => void): void;
+    /**
+     * **connected** & **disconnected** events reflect connection state changes against the (delta)
+     * service acknowledging ops/edits.
+     */
+    (event: "connected" | "disconnected", listener: () => void): void;
+    /**
+     * **saved** event is raised when all local changes/edits have been acknowledged by the service.
+     * **dirty** event is raised when first local change has been made, following a "saved" state.
+     */
+    // eslint-disable-next-line @typescript-eslint/unified-signatures
+    (event: "saved" | "dirty", listener: () => void): void;
+    /**
+     * Disposed event is raised when container is closed. If container was closed due to error
+     * (vs explicit **dispose** action), optional argument contains further details about the error.
+     */
+    (event: "disposed", listener: (error?: ICriticalContainerError) => void);
```

Comment:
Documentation at the top of this file also needs to be updated to "disposed" from "dispose"